packages/flutter_tools/lib/src/commands/upgrade.dart

```diff
@@ -43,6 +43,11 @@ class UpgradeCommand extends FlutterCommand {
     if (code != 0)
       return code;
 
+    // Causes us to update our locally cached packages.
+    code = await runCommandAndStreamOutput(<String>[
+      'bin/flutter', '--version'
```

Comment:
Should we have a flag in the shell script that avoids actually running the Dart part? Or just have all the update logic in a bash script separate from the "flutter" one, so that we can run that directly?